hail/python/test/hail/methods/test_impex.py

```diff
@@ -1136,10 +1143,6 @@ def test_contig_recoding_defaults(self):
             reference_genome='GRCh38',
         )
 
-        rg_random = hl.ReferenceGenome(
-            "random", ['1', '23', '24', '25', '26'], {'1': 10, '23': 10, '24': 10, '25': 10, '26': 10}
-        )
```

Comment:
Is `ruff` deleting this because it's unused? Unfortunately the constructor side-effects and this is actually necessary for the following line with `reference_genome='random'`.

Reply:
all the deletions were me, ruff just removes the variable assignment, didn't realize this has side effects!

Reply:
went through and instead of deleting any unused variables, just deleted the left hand side of the assignments